docs/proposals/20200602-machine-deletion-phase-hooks.md

```diff
@@ -109,7 +109,7 @@ lifecycle hook.
 - Create a mechanism to signal what lifecycle point a machine is at currently.
 - Dictate implementation of controllers that respond to the hooks.
 - Implement ordering in the machine-controller.
-- Require anyone use these hooks for normal machine operations, these are
+- Require anyone uses these hooks for normal machine operations, these are
```

Comment:
I think in this context and format, `use` is preferred over `uses`. It think these bullets are imperative so that would make me lean towards `use` over `uses` anyway

Reply:
I think we should be `Require anyone to use ...`, this is `require...to do something ` grammar

Reply:
`Require anyone to use` seems ok to me